zuul.d/jobs.yaml

```diff
@@ -10,3 +10,29 @@
         CEPH_DATASIZE: "2Gi"
       cifmw_kuttl_tests_operator_list:
         - manila
+
+- job:
+    name: manila-operator-tempest
+    parent: cifmw-base-multinode-tempest
+    attempts: 1
+    vars:
+      cifmw_edpm_prepare_update_os_containers: true
```

Comment:
cifmw_edpm_prepare_update_os_containers and cifmw_set_openstack_containers_* is used to replace the openstack services containers, but your content_provider job is building the manila-operator images. So, you don't need to add these vars here.